artifactory/utils/buildinfoproperties.go

```diff
@@ -214,13 +216,17 @@ func CreateBuildInfoPropertiesFile(buildName, buildNumber, projectKey string, de
 	if err != nil {
 		return "", err
 	}
-	if detailedSummary {
+	if shouldCreateArtifactsFile {
 		err = createDeployableArtifactsFile(config)
 		if err != nil {
 			return "", err
 		}
 	}
 
+	if xrayScan {
```

Comment:
Let's add a comment, explaining why deploy is set to false when scan is enabled.